pkg/bootstrap/bootstrap.go

```diff
@@ -62,8 +62,27 @@ type Options struct {
 	// Issue tracking sync periods per controller: https://github.com/reddit/achilles-sdk/issues/171
 	SyncPeriod time.Duration
 
-	// LeaderElection determines whether the controller should use leader election (a form of active-passive HA).
+	// Determines whether the controller should use leader election (a form of active-passive HA).
 	LeaderElection bool
+
+	// LeaderElectionID determines the name of the resource (Lease) that leader election
+	// will use for holding the leader lock.
+	LeaderElectionID string
+
+	// LeaderElectionNamespace determines the namespace in which the leader
+	// election resource (Lease) will be created.
+	LeaderElectionNamespace string
```

Comment:
I was hoping if we could default this to the application's Namespace, but that would require the user to wire up a downwards API environment variable, which we have no control over in this layer